include/ClientSession.class.php

```diff
@@ -464,13 +464,13 @@ public function clientQuit()
      * @return int
      * @throws ClientSessionException
      */
-    public function setHostVote($host_id, $vote)
+    public function setHostVote(int $host_id, int $vote)
```

Comment:
Does this work on PHP 7.1? :O

Reply:
Sure, scalar typehint were introduced in php 7.0

Reply:
They are mostly missing, I can add them of course.